xbmc/music/Album.h

```diff
@@ -37,7 +37,7 @@ class CAlbum
 {
 public:
   CAlbum(const CFileItem& item);
-  CAlbum() { idAlbum = 0; iRating = 0; iYear = 0; iTimesPlayed = 0; dateAdded.Reset(); lastPlayed.Reset(); releaseType = Album; };
+  CAlbum() { idAlbum = 0; iRating = 0; iYear = 0; iTimesPlayed = 0; dateAdded.Reset(); lastPlayed.Reset(); releaseType = Album; bCompilation = false; };
   bool operator<(const CAlbum &a) const;
```

Comment:
Use proper initializer and not ctor body, also make sure they line up with order of declaration  file to avoid warnings